index.bs

```diff
@@ -553,6 +553,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     cryptographically signing and returning, in the form of an [=Authentication Assertion=], 
     a challenge and other data presented by a [=[WRP]=] (in concert with the [=[WAC]=]).
 
+    A [=[WAA]=] could be, for example, a dedicated USB or wireless device separate from the [=client device=],
+    a dedicated hardware subsystem integrated into the [=client device=],
+    or a software component of the [=client=].
+    [=Authenticators=] can report their type and security characteristics via [=attestation=].
```

Comment:
I believe that, grammatically, "attestation" should be "attestations".

Reply:
I think this sentence can be argued to be using "attestation" as an uncountable noun, as is also done in the [Attestation](https://w3c.github.io/webauthn/#attestation) definition:

>[...] In the WebAuthn context, attestation is employed to attest to the provenance of an authenticator and the data it emits; [...]

Do you agree?

Reply:
Moving ahead with merge per @nadalin's request